compiler/src/iree/compiler/DispatchCreation/test/set_encoding.mlir

```diff
@@ -1,32 +1,36 @@
-// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(util.func(iree-dispatch-creation-set-encoding))" %s | FileCheck %s
+// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(util.func(iree-dispatch-creation-set-encoding))" %s | FileCheck %s --check-prefixes="CHECK-ALL, CHECK"
```

Comment:
It is more common to not wrap `check-prefixes=` with `"`, see https://github.com/search?q=repo%3Airee-org%2Firee%20check-prefixes%3D&type=code